karapace/protobuf/reserved_element.py

```diff
@@ -4,21 +4,23 @@
 """
 # Ported from square/wire:
 # wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/internal/parser/ReservedElement.kt
+from __future__ import annotations
+
 from dataclasses import dataclass
 from karapace.protobuf.kotlin_wrapper import KotlinRange
 from karapace.protobuf.location import Location
 from karapace.protobuf.utils import append_documentation
+from karapace.utils import assert_never
 
 
 @dataclass
 class ReservedElement:
     location: Location
+    values: list[str | int | KotlinRange]
     documentation: str = ""
```

Comment:
Note: Changed order of fields here, because `values` doesn't have a sensible default (inferred by `.to_schema()` lacking the ability to handle `value: None`), and since we need to support Python 3.8, we cannot add `kw_only`.